### PR TITLE
CHANGELOG: add 1.20.0 release

### PR DESCRIPTION
**Pull request by ginglis13**
**Description of changes:**

Changelog for the 1.20.0 release.

**Testing done:**

`cargo make`

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,102 @@
...
+[#3893]: https://github.com/bottlerocket-os/bottlerocket/pull/3893
+[#3894]: https://github.com/bottlerocket-os/bottlerocket/pull/3894
+[#3896]: https://github.com/bottlerocket-os/bottlerocket/pull/3896
+[#3897]: https://github.com/bottlerocket-os/bottlerocket/pull/3897
+[#3898]: https://github.com/bottlerocket-os/bottlerocket/pull/3898
+[#3901]: https://github.com/bottlerocket-os/bottlerocket/pull/3901
+[#3905]: https://github.com/bottlerocket-os/bottlerocket/pull/3905
+[#3906]: https://github.com/bottlerocket-os/bottlerocket/pull/3906
+[#3907]: https://github.com/bottlerocket-os/bottlerocket/pull/3906
+[#3907]: https://github.com/bottlerocket-os/bottlerocket/pull/3907
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
[#3906]: https://github.com/bottlerocket-os/bottlerocket/pull/3906
[#3907]: https://github.com/bottlerocket-os/bottlerocket/pull/3907
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,102 @@
...
+* Add Kubernetes 1.30 variants ([#3859], [#3936])
+* Add container-runtime settings to `aws-k8s-*-nvidia` variants ([#3945])
+* Drop Kubernetes 1.25 Metal and VMware variants ([#3896])
+
+### ECS
+* Update ecs-agent to 1.82.3 ([#3939])
+* Use systemd drop-ins to configure the ECS agent ([#3834])
+
+## Build Changes
+* Update twoliter and the SDK ([#3938], [#3880], [#3885], [#3900])
```
Comment (bcressey) [inline CHANGELOG.md]: `#3880`, `#3900` not in list below

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,102 @@
...
+* Remove libbzip2 ([#3944])
+* Pessimize Rust builds that require the AWS SDK ([#3892])
+* Reduce variant matrix in CI/CD ([#3863])
+* Document package build tools for go dependencies ([#3882])
+* Update Go lints in CI/CD ([#3882])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Update Go lints in CI/CD ([#3884])
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,102 @@
...
+* Out-of-tree build enablement
+  * systemd: use build defaults and kernel parameters for unified cgroups ([#3886], [#3935])
```
Comment (gthao313) [inline CHANGELOG.md:32]: Need to fix the indent.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,102 @@
...
+  * early-boot-config: Use standalone provider binaries to fetch user data ([#3637], [#3890])
+  * logdog: retrieve settings via API client ([#3946])
+  * netdog: remove conditional compilation, add hostname helpers ([#3700], [#3898])
+  * schnauzer: add if_not_null template helper ([#3715])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
  * schnauzer: add if_not_null template helper ([#3838])
```

**Review by larvacea — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,102 @@
...
+* Add setting to configure ntp options ([#3852] thanks @domgoodwin)
+* Include swap utilities ([#3829])
+* Update kernels to 6.1.87, 5.15.156, 5.10.215 ([#3934], [#3930])
+
+## Orchestrator Changes
+
+### Kubernetes
+* Add Kubernetes 1.30 variants ([#3859], [#3936])
+* Add container-runtime settings to `aws-k8s-*-nvidia` variants ([#3945])
+* Drop Kubernetes 1.25 Metal and VMware variants ([#3896])
```
Comment (bcressey) [inline CHANGELOG.md]: nit: swap the ordering here since dropping variants is more significant

File: CHANGELOG.md (hunk 4 above)
Comment (ginglis13) [reply CHANGELOG.md:32]: This is for nested bullets in markdown: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#nested-lists

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,102 @@
...
+  * bootstrap-containers: switch to config file ([#3724])
+  * corndog: switch to config file ([#3715])
+  * prairiedog: switch to config file ([#3713], [#3814], [#3836])
+  * thar-be-updates: switch to config file ([#3721])
+  * updog: use modeled types ([#3901])
+  * kernel: remove variant sensitivity ([#3897], [#3905], [#3932])
+* FIPS enablement
+  * add FIPS report to the API ([#3894])
+  * add release-fips package for FIPS functionality ([#3893])
+  * build Go binaries for FIPS and non-FIPS ([#3887])
```
Comment (bcressey) [inline CHANGELOG.md:48]: Would be good to verify (via https://github.com/joeyespo/grip or similar) that this is the right syntax for nested bullet lists (I tend to mess this up)

File: CHANGELOG.md (hunk 2 above)
Comment (ginglis13) [reply CHANGELOG.md]: Ah I see why I missed those - We've already included them in 1.19.5. I'll remove them

File: CHANGELOG.md (hunk 7 above)
Comment (larvacea) [reply CHANGELOG.md:48]: I verified this by previewing the file in GitHub. My eyeballs report: yes, indented bullet list.

Comment (ginglis13) [reply CHANGELOG.md:48]: vscode has a nifty shortcut `<super>+shift+v` to preview which I used.

**Comment by ginglis13** [comment]
^ address comments

File: CHANGELOG.md (hunk 7 above)
Comment (ginglis13) [reply CHANGELOG.md:48]: >  I verified this by previewing the file in GitHub. My eyeballs report: yes, indented bullet list.

TIL! https://github.com/bottlerocket-os/bottlerocket/blob/3152f474fa15ca7c93cf8421a9c533ea9b905f56/CHANGELOG.md

**Review by bcressey — APPROVED** [review]

**Review by yeazelm — APPROVED** [review]

**Review by arnaldo2792 — APPROVED** [review]

**Pull request merged**